src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -253,220 +140,174 @@ If, however, you expect users to interact with multiple elements at the same
 time (using multi-touch), you should restrict the touch to the specific
 element.
 
-<div class="clearfix"></div>
-
+This is more flexible for users, but complicates the logic for manipulating
+the UI and is less resilient to user error.
 
-### TL;DR {: .hide-from-toc }
-- For full device support, handle touch, mouse and Pointer Events.
-- Always bind start event listeners to the element itself.
-- If you want the user to interact with one particular element, bind your move and end listeners to the document in the touchstart method; ensure you unbind them from the document in the end listener.
-- If you want to support multi-touch, either restrict move and end touch events to the element itself or handle all the touches on an element.
+<div class="clearfix"></div>
 
+### Add event listeners
 
-### Add Event Listeners
+In Chrome (version 55 and later) and Internet Explorer + Edge `PointerEvents` are
```

Comment:
Internet Explorer !== Edge and Edge !== Internet Explorer. Both from Microsoft, but seperate.

Pointer Events are in both Internet explorer and Edge.